packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/aspnet/remotesignout.md

```diff
@@ -22,50 +22,4 @@ Update your `using` statements to import the following namespaces:
 ```csharp
 using Microsoft.Owin.Security.Cookies;
 using Okta.AspNet;
```

Comment:
Should we mention that after users sign out of Okta they will be redirected to the location defined in _Define your sign out callback_?